app/ai/voice/agents/breeze_buddy/agent/__init__.py

```
@@ -644,6 +651,44 @@ async def on_client_ready(rtvi):
+                        f"[VOICE_REDIRECT] Intercepted restricted tool '{fc.function_name}' "
+                        f"in voice mode, emitting redirect event. "
+                        f"conversation_id={self.conversation_id}"
+                    )
+
+                    # Speak the redirect message via TTS
+                    if self.task:
+                        try:
+                            await self.task.queue_frame(
+                                TTSSpeakFrame(text=VOICE_REDIRECT_TTS_MESSAGE)
+                            )
+                        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)